### PR TITLE
06-comments

### PR DESCRIPTION
**Pull request by Amir-Rsh**

### CODE REVIEW

File: src/Components/ArticlePage.jsx [hunk 1]
```diff
@@ -15,12 +16,12 @@ export default function ArticlePage({}) {
   }, []);
 
   return isLoading ? (
-    <h6>Article is loading</h6>
+    <p></p>
```
Comment (pippaamy) [inline src/Components/ArticlePage.jsx:19]: wait bring loading back !!

File: src/Components/Comments.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import { getComments } from "../../api";
+
+export default function Comments({ article_id }) {
+  const [commentList, setCommentList] = useState([]);
+  useEffect(() => {
```
Comment (pippaamy) [inline src/Components/Comments.jsx:6]: Im not seeing this page on the article page hmm

**Review by pippaamy — COMMENTED** [review]
I cant even see the header of "Comments:" so I think its not coming through somewhere :/

File: src/Components/ArticlePage.jsx (hunk 1 above)
Comment (Amir-Rsh) [reply src/Components/ArticlePage.jsx:19]: Hi Pippa, there is an isLoading on the app.jsx as well and when i set the is loading on the article page to give any massage it causes a double isloading message. I need to fix that

File: src/Components/Comments.jsx (hunk 2 above)
Comment (Amir-Rsh) [reply src/Components/Comments.jsx:6]: Oh no what have i done 😅 could it be because of my styling. Because it shows up on my screen

**Pull request merged**